internal/controllers/core/cmd/controller.go

```diff
@@ -224,6 +243,15 @@ func (c *Controller) reconcile(ctx context.Context, name types.NamespacedName) e
 //
 // Blocks until the command is finished, then returns its status.
 func (c *Controller) ForceRun(ctx context.Context, cmd *v1alpha1.Cmd) (*v1alpha1.CmdStatus, error) {
+	disableStatus, err := configmap.MaybeNewDisableStatus(ctx, c.client, cmd.Spec.DisableSource, cmd.Status.DisableStatus)
+	if err != nil {
+		return nil, err
+	}
+	if disableStatus.Disabled {
+		// TODO(matt) have the build controller ensure we don't reach this point on disabled commands (ch12734)
```

Comment:
hmmmm...this comment makes it sound like we shouldn't check this in ForceRun at all? The name "ForceRun()" implies to me that we've already done the necessary checks. this seems premature and likely to bite us with race conditions later.